torch/csrc/jit/passes/quantization_patterns.h

```diff
@@ -127,6 +127,18 @@ graph(%packed_params, %a_quant, %r_scale, %r_zero_point, %r_dtype):
         %r = quantized::linear(%a_quant, %packed_params, %r_scale, %r_zero_point)
         return (%r) )";
 
+  std::string cat = R"(
```

Comment:
This needs to work with a list of tensors and for lists of arbitrary length (known). Please add a test.